NEWS.md

```diff
@@ -79,7 +79,7 @@ Standard library changes
 
 * The `init` keyword for `reduce` and other reduction functions without guaranteed
```

Comment:
also "keyword" should be "keyword argument" I think. It's pretty much implied when I went through the PR. But First glance at first statement made me think this is introducing new keyword.